packages/components/autocomplete/stories/autocomplete.stories.tsx

```diff
@@ -1061,3 +1081,10 @@ export const FullyControlled = {
     ...defaultProps,
   },
 };
+
+export const WithAutoHighlight = {
+  render: AutoHighlightTemplate,
```

Comment:
there's no need to create a new template for that. You can simply use `Template` one and just pass `autoHighlight` to args below.